osu.Game/Localisation/AccountCreationStrings.cs

```diff
@@ -0,0 +1,72 @@
+
+        /// <summary>
+        /// "New Player Registration"
+        /// </summary>
+        public static LocalisableString NewPlayerRegistration => new TranslatableString(getKey(@"new_player_registration"), @"New Player Registration");
+
+        /// <summary>
+        /// "let&#39;s get you started"
+        /// </summary>
+        public static LocalisableString LetsGetYouStarted => new TranslatableString(getKey(@"lets_get_you_started"), @"let's get you started");
+
+        /// <summary>
+        /// "Let&#39;s create an account!"
+        /// </summary>
+        public static LocalisableString LetsCreateAnAccount => new TranslatableString(getKey(@"lets_create_an_account"), @"Let's create an account!");
+
+        /// <summary>
+        /// "Help, I can&#39;t access my account!"
+        /// </summary>
+        public static LocalisableString HelpICantAccess => new TranslatableString(getKey(@"help_icant_access"), @"Help, I can't access my account!");
+
+        /// <summary>
+        /// "I understand. This account isn&#39;t for me."
+        /// </summary>
+        public static LocalisableString AccountIsntForMe => new TranslatableString(getKey(@"account_isnt_for_me"), @"I understand. This account isn't for me.");
+
+        /// <summary>
+        /// "This will be your public presence. No profanity, no impersonation. Avoid exposing your own personal details, too!"
+        /// </summary>
+        public static LocalisableString ThisWillBeYourPublic => new TranslatableString(getKey(@"this_will_be_your_public"),
+            @"This will be your public presence. No profanity, no impersonation. Avoid exposing your own personal details, too!");
+
+        /// <summary>
+        /// "Will be used for notifications, account verification and in the case you forget your password. No spam, ever."
+        /// </summary>
+        public static LocalisableString EmailUsage =>
+            new TranslatableString(getKey(@"email_usage"), @"Will be used for notifications, account verification and in the case you forget your password. No spam, ever.");
+
+        /// <summary>
+        /// " Make sure to get it right!"
+        /// </summary>
+        public static LocalisableString MakeSureToGetIt => new TranslatableString(getKey(@"make_sure_to_get_it"), @" Make sure to get it right!");
+
+        /// <summary>
+        /// "At least "
+        /// </summary>
+        public static LocalisableString BeforeCharactersLong => new TranslatableString(getKey(@"before_characters_long"), @"At least ");
+
+        /// <summary>
+        /// "8 characters long"
+        /// </summary>
+        public static LocalisableString CharactersLong => new TranslatableString(getKey(@"characters_long"), @"8 characters long");
+
+        /// <summary>
+        /// ". Choose something long but also something you will remember, like a line from your favourite song."
+        /// </summary>
+        public static LocalisableString AfterCharactersLong =>
```

Comment:
This won't work well for languages that aren't english. You'll need to use a formatting string with the "8 characters long" part as an argument.

Reply:
Are there existing examples how this should work?
I found this case which is quite similar, but it works with links, not custom drawables:

https://github.com/ppy/osu/blob/3f2077a527ade1380b77c9ac55ca2ec3ddd84616/osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs#L112-L117

I tried with splitting the string on place, but it looks ugly.

Reply:
I don't believe there are any. Also, the code above doesn't even work, `FormatText` and `GetLocalisedBindableString().Value` return a string, so the `LocalisableString` fallbacks to English.

Reply:
Am i right that we need some kind of `TextFlowContainer` which will track base localisable string with custom drawables inside it and recompose the text on language changes?

Reply:
I would not use `CustomizableTextContainer` here. At least not raw. If I'm thinking correctly what you were going to try and do (replace a placeholder with a `SpriteText`), it will not handle line breaking correctly.

Needs proper framework support somehow. I don't yet know how.